requirements.txt

```diff
@@ -0,0 +1,6 @@
+py-ecc==6.0.0
```

Comment:
@qwang98 @trangnv should this be in the root folder or in src/frontend/python?

Reply:
Technically the entire requirements.txt is for Python. I think putting them at the root folder is easier for users to call `pip install -r requirements.txt`, but let me know if you think otherwise.

Reply:
Ok